### ISSUE TITLE
LTR insertion time

### ISSUE THREAD

**Issue by hlin1221** [oushujun/LTR_retriever#182]
The LTR insertion time used intact to calculate the insertion. I wonder weather this affect the real result. I think that intact may have a short insertion time, while fragmented have a long insertion time.

**Comment by oushujun** [comment]
Yes, it only reflects intact LTR insertion time spectrum, not all LTRs
because the challenge in inferring the full-length structure of LTRs
increases exponentially as they aged.

Shujun
